privacyidea/lib/importotp.py

```diff
@@ -455,6 +455,7 @@ def derive_key(xml, password):
 def parsePSKCdata(xml_data,
                   preshared_key_hex=None,
                   password=None,
+                  validate_mac='check_fail_hard',
```

Comment:
Just one thought, sometimes it might be safer to add a new parameter at the end of the parameter list in case there is any call to this function, that does not specify the keywords, like a call to:

    parsePKSCdata(data, None, "secret", True)

when the caller wants to check the serials. 
(No need to change this, just for information)